packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -89,16 +93,37 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
         },
         panelContent: {
             [`& .${gridClasses.filterForm}:first-child .${gridClasses.filterFormLinkOperatorInput}`]: {
```

Comment:
Please avoid using `:first-child` as this causes the following error message in the console: 

```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".
```